src/components/collection/Config.tsx

```diff
@@ -1,88 +1,36 @@
 import { useRouteStore } from 'hooks/useRouteStore';
-import { useEffect } from 'react';
 import { FormattedMessage } from 'react-intl';
-import { useSearchParams } from 'react-router-dom';
 import { entityCreateStoreSelectors } from 'stores/Create';
-import { ENTITY } from 'types';
 
 function CollectionConfig() {
     const useEntityCreateStore = useRouteStore();
     const imageTag = useEntityCreateStore(
         entityCreateStoreSelectors.details.connectorTag
     );
-    const collections = useEntityCreateStore(
-        entityCreateStoreSelectors.collections
-    );
-    const prefillCollections = useEntityCreateStore(
-        entityCreateStoreSelectors.prefillCollections
-    );
-
-    const [searchParams] = useSearchParams();
-    const specID = searchParams.get(
-        authenticatedRoutes.materializations.create.params.liveSpecId
-    );
-    const lastPubId = searchParams.get(
-        authenticatedRoutes.materializations.create.params.lastPubId
-    );
-
     const resourceConfigHasErrors = useEntityCreateStore(
         entityCreateStoreSelectors.resourceConfig.hasErrors
     );
     const collectionsHasErrors = useEntityCreateStore(
-        entityCreateStoreSelectors.collectionsHasErrors
-    );
-
-    const { liveSpecs } = useLiveSpecsExtWithOutSpec(specID, ENTITY.CAPTURE);
-    const { liveSpecs: liveSpecsByLastPub } = useLiveSpecsExtByLastPubId(
-        lastPubId,
-        ENTITY.CAPTURE
+        entityCreateStoreSelectors.collections.hasErrors
     );
-
-    useEffect(() => {
-        if (liveSpecs.length > 0) {
-            prefillCollections(liveSpecs);
-        }
-    }, [liveSpecs, prefillCollections]);
-
-    useEffect(() => {
-        if (liveSpecsByLastPub.length > 0) {
-            prefillCollections(liveSpecsByLastPub);
-        }
-    }, [liveSpecsByLastPub, prefillCollections]);
+    const hasErrors = resourceConfigHasErrors || collectionsHasErrors;
 
     if (imageTag) {
         return (
             <WrapperWithHeader
                 header={
                     <>
-                        {resourceConfigHasErrors || collectionsHasErrors ? (
+                        {hasErrors ? (
                             <ErrorOutlineIcon color="error" sx={{ pr: 1 }} />
                         ) : null}
                         <FormattedMessage id="materializationCreate.collections.heading" />
                     </>
                 }
             >
-                <>
-                    <CollectionSelector />
-                    {collections.map((collection: any, index: number) => {
-                        return (
-                            <ExpandableResourceConfig
-                                collectionName={collection}
-                                id={imageTag.id}
-                                key={`CollectionResourceConfig-${index}`}
-                            />
-                        );
-                    })}
-                </>
+                <BindingsMultiEditor />
```

Comment:
New collections editor... calling it bindings editor since it can be for any bindings.